pkg/gadgets/run/types/types.go

```diff
@@ -50,8 +56,11 @@ type Event struct {
 	MountNsID uint64 `json:"-"`
 	NetNsID   uint64 `json:"-"`
 
-	// Raw event sent by the ebpf program
-	RawData []byte `json:"raw_data,omitempty"`
+	// Blob is used to save data to be sent to the client.
+	// [0] is used for bpf event
+	// [1] is used for fixed-size members
+	// [1+] is used for variable size members
+	Blob [][]byte `json:"blob,omitempty"`
```

Comment:
I find the indexing system (IndexVirtual, IndexEBPF, IndexFixed) confusing. Would the following make the code clearer?

```suggestion
	// Blobs used to save data to be sent to the client.
	BpfEvent []byte `json:"bpfEvent,omitempty"`
	FixedSizedMembers []byte `json:"fixedSizeMembers,omitempty"`
	VariableSizedMembers [][]byte `json:"fixedSizeMembers,omitempty"`
```